### PR TITLE
PackageTracking - Merge all package tracking IA's into 1

### PR DESCRIPTION
**Pull request by Mailkov**
This is WIP

This IA uses api: https://api.packagetrackr.com/docs/v1/track/simple

Fixes https://github.com/duckduckgo/zeroclickinfo-goodies/issues/485
Fixes https://github.com/duckduckgo/zeroclickinfo-goodies/issues/1138
Fixes https://github.com/duckduckgo/zeroclickinfo-goodies/issues/1828
Fixes https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2845

---
IA page: https://duck.co/ia/view/package_tracking

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/spice/package_tracking/package_tracking.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                subtitle: api_result.location,
+                description: "Last update: " + DDG.getDateFromString(api_result.progress_at),
+                image: DDG.get_asset_path('package_tracking', 'logos/' + logo + '.png'),
+                infoboxData: [
+                    {
+                        heading: "Info shipping"
+                    },
+                    {
+                        label: "Shipped at: ",
+                        value: api_result.shipped_at != null ? DDG.getDateFromString(api_result.shipped_at) : ""
```
Comment (moollaza) [inline share/spice/package_tracking/package_tracking.js]: Let's use `Moment.js` so we can format these in a way that's easier to read: i.e. `Jan, 1st, 2016`

We can update this to `value: (api_result.shipped_at) ? moment.format('MMM DD, YYYY') : "",`

**Comment by zekiel** [comment]
Just to note, it doesn't seem like the international DHL packages work per issue https://github.com/duckduckgo/zeroclickinfo-goodies/issues/642

**Comment by moollaza** [comment]
@Mailkov it looks like you've rebased this incorrectly I'm a little uneasy to merge this.

Can you try correcting the pull request and also move the logos into the root of the /share/ directory? I'm still unsure why the carrier logos aren't working correctly. 

After that I think we'll have something acceptable for a V1 :+1:

**Comment by jdorweiler** [comment]
This pull request is being closed due to inactivity but, if anyone would like to continue it, please @ mention 'duckduckhack-contributors' for help and http://docs.duckduckhack.com/submitting/submitting-overview.html for reference.

**Comment by Mailkov** [comment]
@jdorweiler please can you reopen it? Thanks

**Comment by jdorweiler** [comment]
Done!

**Comment by moollaza** [comment]
@Mailkov apologies for the delay here! This is now live and working on Beta. It looks like you have a JS error though. Once that's sorted I think this is just about done!

File: share/spice/package_tracking/package_tracking.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                    subtitle: api_result.location,
+                    description: "Last update: " + DDG.getDateFromString(api_result.progress_at),
+                    image: DDG.get_asset_path('package_tracking', logo + '.png'),
+                    infoboxData: [
+                        {
+                            heading: "Info shipping"
+                        },
+                        {
+                            label: "Shipped at: ",
+                            value: (api_result.shipped_at) ? moment.format('MMM DD, YYYY') : ""
```
Comment (moollaza) [inline share/spice/package_tracking/package_tracking.js]: You need to hand the string/object to `moment()` and then chain the `.format()` method after:

```js
value: (api_result.shipped_at) ? moment(api_result.shipped_at).format('MMM DD, YYYY') : ""

**Comment by moollaza** [comment]
Continued on a new branch: https://github.com/duckduckgo/zeroclickinfo-spice/tree/pr/2430

**Pull request closed without merging**